lib/atlas/graph_deserializer.rb

```diff
@@ -0,0 +1,72 @@
+module Atlas
+  class GraphDeserializer
+    def self.build(graph_hash)
```

Comment:
:+1:

(Sorry for these emoji only remarks. The mobile version does not seem to allow the direct emoji reactions)